### ISSUE TITLE
Remove the "Your homepage displays" setting from Reading Settings, when an FSE-ready theme with a Home template is active

### ISSUE THREAD

**Issue by danielk-am** [Automattic/wp-calypso#69163]
### What

I would like to request to remove the "Your homepage displays" setting from the Reading Settings when an FSE-ready theme is active.

### Why

This is to simplify the way users can work on their Home template, without the need to select "another" setting that is outside the Site Editor.

**Comment by zachspears** [comment]
Hello @1dr0, Currently, this setting is used to define which page is shown as the homepage or provide the option to have a dynamic blog post page depending on the theme. Are you proposing moving this setting to somewhere within FSE itself?

**Comment by danielk-am** [comment]
Howdy! I am proposing to remove the setting altogether or to force it to display the latest posts only when an FSE-ready theme with a Home template is active.

This makes the Home template a lot more intuitive. Where I would expect the customizations done to the Home template to appear on the homepage, without having to check if I had set the Reading Setting's homepage display correctly or not.

Right now, setting the Reading Setting to display a static home page meant that the Home template becomes the Posts template, which may confuse users.

<img width="975" alt="Screen Shot 2022-10-22 at 14 28 28" src="https://user-images.githubusercontent.com/92185881/197324023-c7676d3f-f463-4824-8c3b-89f739f22d59.png">

```
Home 
Displays posts on the homepage, or on the Posts page if a static homepage is set.
```

The above example was from a user's site that has Archeo as the active theme. They had spent some time trying to customize their Home template without getting it to work the way they wanted until I checked that they had set that they had their settings set to display a static home page, so the Home template would not work.

37473936-hc

---

Meanwhile, it appears that not all themes come with a Home template. I've updated the title to narrow this to FSE-ready themes with a Home template. I hope the above clarifies!
